README.md

```diff
@@ -38,15 +37,23 @@ However, you may want to use a different version of cesium, like if you've clone
 
 	var cesiumSource = path.resolve(__dirname, '../path/to/cesium/Source');
 
-You could even point the cesium to a branch in github or another url in your `package.json` file.
+You could even point the cesium to a branch in GitHub or another url in your `package.json` file.
+
+### Source Maps
```

Comment:
Should we also mention SourceMapDevToolPlugin?

Reply:
The documentation I linked to above links to that plugin. Unless it requires some specific configuration in order to work with Cesium, I'd say we should leave it out.

Reply:
Ooops! My bad! You are right, I did not notice it :smirk: